lib/util/lib/workqueue.c

```diff
@@ -639,14 +634,30 @@ begin_stats_work(void)
 
 enum rest_status
 rest_get_workqueues(
-    const struct rest_request *const req,
-    struct rest_response *const resp,
-    void *const arg)
+    const struct rest_request * const req,
+    struct rest_response * const resp,
+    void * const arg)
 {
-    static const ulong divtab[] = {
-        1, 0, 1, 1, 1e3, 10, 1e6, 10, 1e9, 10, 1e9 * 3600, 2, 1e9 * 86400, 1,
-        1e9 * 86400 * 365, 1, 1e9 * 86400 * 365 * 100, 1, ULONG_MAX, 1
-    };
+    static const ulong divtab[] = { 1,
+                                    0,
+                                    1,
+                                    1,
+                                    1e3,
+                                    10,
+                                    1e6,
+                                    10,
+                                    1e9,
+                                    10,
+                                    1e9 * 3600,
+                                    2,
+                                    1e9 * 86400,
+                                    1,
+                                    1e9 * 86400 * 365,
+                                    1,
+                                    1e9 * 86400 * 365 * 100,
+                                    1,
+                                    ULONG_MAX,
+                                    1 };
```

Comment:
I guess we have fix all these by hand, do the makers of clang-format really think this is awesome stuff?

Reply:
The beauty of clang-format is that there are a lot of toggles, so presumably one of our toggles is causing this.

Reply:
Actually, clang-format has about 1% of the "toggles" of most formatters I've seen/used in the past.  And that's the problem I have with it, it's super-inflexible and does a lot of really dumb stuff...